app/grandchallenge/challenges/emails.py

```diff
@@ -91,3 +91,24 @@ def send_challenge_status_update_email(challengerequest, challenge=None):
         recipients=[challengerequest.creator],
         subscription_type=EmailSubscriptionTypes.SYSTEM,
     )
+
+
+def send_email_percent_budget_consumed_alert(challenge, warning_threshold):
+    send_standard_email_batch(
+        site=Site.objects.get_current(),
+        subject=format_html(
+            "[{challenge_name}] Challenge {warning_threshold}% Budget Consumed Alert",
```

Comment:
I think Challenge is redundant here, right? 

```suggestion
            "[{challenge_name}] {warning_threshold}% Budget Consumed Alert",
```